client/app/admin/create-project/create-project.controller.js

```diff
@@ -236,8 +239,21 @@
             }, function (reason) {
                 vm.waiting = false;
                 vm.trimError = true;
+
+                switch (reason.status) {
+                    case 400:
+                        vm.trimErrorReason = reason.data.error;
+                        break;
+                    case 500:
+                        vm.trimErrorReason = reason.data.error;
+                        break;
+                    default:
+                        vm.trimErrorReason = 'The grid could not be trimmed. This may be because the combined data size of the AOI and the ' +
+                        'grid is too large. Try again with a smaller AOI, fewer tasks and/or larger tasks.'
```

Comment:
We can't have any text in the controller. This would make translation difficult...We also can't use any error messages coming from the server.